libs/types/src/cast_vote_records.ts

```diff
@@ -101,3 +115,36 @@ export type SheetValidationError = {
 export type ExportCastVoteRecordsToUsbDriveError =
   | { type: ExportDataError }
   | SheetValidationError;
+
+/**
+ * An error encountered while reading a cast vote record export's metadata file
+ */
+export type ReadCastVoteRecordExportMetadataError =
+  | { type: 'metadata-file-not-found' }
+  | { type: 'metadata-file-parse-error' };
+
+/**
+ * A top-level error encountered while reading a cast vote record export. Does not include errors
+ * encountered while reading individual cast vote records.
+ */
+export type ReadCastVoteRecordExportError =
+  | ReadCastVoteRecordExportMetadataError
+  | { type: 'authentication-error' };
+
+type ReferencedFile = 'image' | 'layout-file';
+
+/**
+ * An error encountered while reading an individual cast vote record
+ */
+export type ReadCastVoteRecordError = { type: 'invalid-cast-vote-record' } & (
```

Comment:
This is the one type that I've actually tweaked a bit, adding some additional error `subType` values, which you'll see get used in subsequent commits